From 462dff237f6e8e3c339648321d08686b06194e1c Mon Sep 17 00:00:00 2001
From: Vitaly Andrianov <vitalya@ti.com>
Date: Mon, 20 Feb 2017 12:48:28 -0500
Subject: [PATCH 93/94] am57xx: jailhouse:reserve 2 GIC interrupt lines for
 inmate

We are going top use shared peripheral interrupters in an imates. For that,
we have to reserve some of GICD interrupt input lines not to be used by
crossbar manager.

This commit reserves GICD lines #134 and #135.

Signed-off-by: Vitaly Andrianov <vitalya@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/boot/dts/dra7.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/dra7.dtsi b/arch/arm/boot/dts/dra7.dtsi
index 3d8ada4..b560933 100644
--- a/arch/arm/boot/dts/dra7.dtsi
+++ b/arch/arm/boot/dts/dra7.dtsi
@@ -2014,7 +2014,7 @@
 			ti,max-crossbar-sources = <MAX_SOURCES>;
 			ti,reg-size = <2>;
 			ti,irqs-reserved = <0 1 2 3 5 6 131 132>;
-			ti,irqs-skip = <10 133 139 140>;
+			ti,irqs-skip = <10 133 134 135 139 140>;
 			ti,irqs-safe-map = <0>;
 		};
 
-- 
1.9.1

